tests/templates/kuttl/ldap-authentication/03-openldap-secret-class.yaml

```diff
@@ -0,0 +1,23 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+metadata:
+  name: openldap
+commands:
+  # SecretClass requires an explicit namespace to work
+  - script: |
+      kubectl apply -n $NAMESPACE -f - <<EOF
+      ---
+      apiVersion: secrets.stackable.tech/v1alpha1
+      kind: SecretClass
+      metadata:
+        name: openldap-tls
+      spec:
+        backend:
+          autoTls:
+            ca:
+              autoGenerate: true
+              secret:
+                name: openldap-tls-ca
+                namespace: $NAMESPACE
+      EOF
```

Comment:
Nitpicking... Have a look at this: https://github.com/stackabletech/trino-operator/blob/main/tests/templates/kuttl/ldap/00-install-openldap.yaml
I see why you split it up (it basically saves one "$NAMESPACE" :D) 
Although if we ever have to customize here id prefer to do it in the install_openldap.yaml.j2.

Reply:
We could move it, to be similar to the pattern in the link. I don't see a big upside to either option.

Reply:
Yeah lets keep it as is.